Cargo.toml

```diff
@@ -108,21 +108,27 @@ strum_macros = "0.24"
 backtrace = "0.3"
 linkme = "0.3"
 serde = { version = "1", features = ["derive"] }
+cfg-if = "1"
+redis-module-derive-internals = { path = "./redismodule-rs-derive-internals"}
```

Comment:
Only the internal one? or also the `redis-module-derive`. Asking because we need to decide this before we release the next version. I tends to say that `derive` is more common no? Like `serde-derive`.

Reply:
OK, I will rename it on this PR before we will release the next version.

Reply:
@vityafx done